src/app/components/Feedback/Feedback.jsx

```diff
@@ -0,0 +1,194 @@
+
+  render() {
+    const showForm = this.state.showForm;
+
+    return (
+      <div className="feedback">
+        <button
+          className="feedback-button"
+          onClick={() => this.openForm()}
+          aria-haspopup="true"
+          aria-expanded={showForm}
+          aria-controls="feedback-menu"
+        >
+          Feedback
+        </button>
+        <FocusTrap
+          focusTrapOptions={{
+            onDeactivate: this.deactivateForm,
+            clickOutsideDeactivates: true,
+          }}
+          active={showForm}
+        >
+          <div
+            role="menu"
+            className={`feedback-form-container${showForm ? ' active' : ''}`}
+            id="feedback-menu"
+          >
+            <form onSubmit={e => this.onSubmitForm(e)}>
+              <div>
+                <label id="sfr-feedback-success">Did you find what you were looking for?</label>
+                <div>
+                  <input 
+                    type="radio"
+                    className="sfr-feedback-radio"
+                    id="sfr-feedback-found-yes"
+                    name="feedback"
+                    value="yes"
+                    onChange={this.handleRadioChange}
+                  />
+                  <label htmlFor="sfr-feedback-found-yes" className="sfr-radio-label">Yes</label>
+                </div>
+
+                <div>
+                  <input
+                    type="radio"
+                    className="sfr-feedback-radio"
+                    id="sfr-feedback-found-no"
+                    name="feedback"
+                    value="no"
+                    onChange={this.handleRadioChange}
+                  />
+                  <label htmlFor="sfr-feedback-found-no" className="sfr-radio-label">No</label>
+                </div>
+
+                <div>
+                  <input
+                    type="radio"
+                    className="sfr-feedback-radio"
+                    id="sfr-feedback-found-browse"
+                    name="feedback"
+                    value="browse"
+                    onChange={this.handleRadioChange}
+                  />
+                  <label htmlFor="sfr-feedback-found-browse" className="sfr-radio-label">Just Browsing</label>
+                </div>
+              </div>
+              <div>
+                <label htmlFor="feedback-textarea-comment">
+                  Comments (Required)
+                </label>
+                <textarea
+                  id="feedback-textarea-comment"
+                  className="feedback-input"
+                  name="sfr-general-feedback"
+                  rows="5"
+                  aria-required="true"
+                  tabIndex="0"
+                  ref={(textarea) => this.feedbackField = textarea}
+                  value={this.state.feedback}
+                  onChange={this.handleFeedbackChange}
+                />
+                <div id="textarea-status-note">{this.state.commentStatus}</div>
```

Comment:
you want `{ this.state.commentStatus && <div id="textarea-status-note">Comment Required</div>}` here
(Conventions - don't want to render this div if you don't have to)